src/lib/auth.ts

```diff
@@ -0,0 +1,13 @@
+import { createSaleorAuthClient } from "@saleor/auth-sdk";
+import { getNextServerCookiesStorage } from "@saleor/auth-sdk/next/server";
+import { invariant } from "ts-invariant";
+
+const nextServerCookiesStorage = getNextServerCookiesStorage();
+
+invariant(process.env.NEXT_PUBLIC_SALEOR_API_URL, "Missing NEXT_PUBLIC_SALEOR_API_URL env variable");
+
+export const authClient = createSaleorAuthClient({
```

Comment:
We could reuse the instance created in graphql.ts and then delete thus file

Reply:
@mmiszy yeah, I wanted to move `<Provider>` into checkout's `layout.tsx` and clearly separate things

Reply:
Is this done?